.gitignore

```diff
@@ -6,3 +6,4 @@ y2signal.log
 coverage/
 fisx-me
 sap-installation-wizard.pot
+RUBOKOP
```

Comment:
Interesting, I never see such file. What creates it? Some IDE?

Reply:
By  rewriting the code I always created the output of rubokop in this file. This was very long at the beginig. More then 3000 lines. To awoid that git status concerns I've introduced it. 
Respect! But nothing escapes you either.